alienfile

```diff
@@ -0,0 +1,20 @@
+use alienfile;
+
+plugin 'Probe::CommandLine' => (
+  command   => 'gpg-error',
+  secondary => 1,
+);
+
+share {
+
+  plugin Download => (
+    url     => 'https://gnupg.org/download/index.html#libgpg-error',
```

Comment:
Although my preference is usually to download rather than bundle, we could keep this as bundled if @ajgb  prefers it.  If so I would move the tarball into the patch directory (so that it works with `App::af` rebuilds).  There is an example in the synopsis of:  https://metacpan.org/pod/Alien::Build::Plugin::Fetch::Local